### ISSUE TITLE
can't select one/two balloon toggle buttons via keyboard navigation

### ISSUE THREAD

**Issue by jbphet** [phetsims/balloons-and-static-electricity#259]
In version http://www.colorado.edu/physics/phet/dev/html/balloons-and-static-electricity/1.3.0-dev.1/balloons-and-static-electricity_en.html the one/two toggle button cannot be accessed via keyboard navigation.  This was tested by pressing the tab key and cycling through all selectable objects, and it looks like this control is just skipped.

This was originally reported by @emily-phet in https://github.com/phetsims/balloons-and-static-electricity/issues/240#issuecomment-296201126.

**Comment by jessegreenberg** [comment]
This is now a duplicate of #264
